regions_www/m/ui.R

```diff
@@ -77,18 +77,15 @@ shinyUI(
           includeHTML(file.path("..", "favicon.html"))
         ),
         includeHTML(file.path("..", "www", "test-banner.html")),
-        br(),
         div(id="loading", "Loading&#8230;"),
-        leafletOutput("map", width = "100%", height = "95%"),
         absolutePanel(
           id = "controls",
           class = "panel panel-default",
           fixed = TRUE,
-          top = 110,
           right = 20,
           width = 180,
           height = "auto",
-          style = "opacity: 0.9",
+          style = "opacity: 0.9; z-index: 1; position: absolute",
```

Comment:
Great change, this is very clever stuff @nikolai-b. I cannot seen any negative side effects.